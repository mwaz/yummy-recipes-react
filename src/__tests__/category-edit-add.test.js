import React from 'react';
import { shallow, mount, render } from 'enzyme';
import ReusableModal from '../components/common/category-edit-add-modal';
import toJson, { shallowToJson } from 'enzyme-to-json';

describe('ReusableModal component', () => {
    const modals = jest.fn();
    const wrapper = shallow(<ReusableModal modals={modals} location={{}} />);
    const preventDefault = jest.fn();

    it('renders properly without crashing', () => {
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });

    

});