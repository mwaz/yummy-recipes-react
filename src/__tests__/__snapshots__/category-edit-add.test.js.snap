// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReusableModal component renders properly without crashing 1`] = `
<Modal
  animation={true}
  autoFocus={true}
  backdrop={true}
  bsClass="modal"
  className="modal-fade"
  dialogComponentClass={[Function]}
  enforceFocus={true}
  keyboard={true}
  manager={
    ModalManager {
      "add": [Function],
      "containers": Array [],
      "data": Array [],
      "handleContainerOverflow": true,
      "hideSiblingNodes": true,
      "isTopModal": [Function],
      "modals": Array [],
      "remove": [Function],
    }
  }
  onHide={[Function]}
  renderBackdrop={[Function]}
  restoreFocus={true}
  show={false}
>
  <form>
    <ModalHeader
      bsClass="modal-header"
      closeButton={false}
      closeLabel="Close"
    >
      <ModalTitle
        bsClass="modal-title"
        componentClass="h4"
      >
         
          
      </ModalTitle>
    </ModalHeader>
    <ModalBody
      bsClass="modal-body"
      componentClass="div"
    >
      <div
        className="form-group"
      >
        <label
          className="col-md-4 control-label"
          htmlFor="Category_name"
        >
          Category Name
        </label>
        <div
          className="col-md-8"
        >
          <input
            className="form-control"
            id="new_cat"
            type="text"
          />
        </div>
      </div>
    </ModalBody>
    <ModalFooter
      bsClass="modal-footer"
      componentClass="div"
    >
      <InputGroupButton
        bsClass="input-group-btn"
      >
        <Button
          active={false}
          block={false}
          bsClass="btn"
          bsStyle="info"
          disabled={false}
          type="submit"
        >
           
           
        </Button>
      </InputGroupButton>
      <Button
        active={false}
        block={false}
        bsClass="btn"
        bsStyle="danger"
        disabled={false}
      >
         Cancel
      </Button>
    </ModalFooter>
  </form>
</Modal>
`;
